src/main/java/seedu/address/ui/StudentCard.java

```diff
@@ -48,7 +48,23 @@ public StudentCard(Student student, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(student.getName().fullName);
         email.setText("Email: " + student.getEmail().value);
-        tag.getChildren().add(new Label(student.getTag().ranking.toString()));
+        String studentTag = student.getTag().ranking.toString();
+        Label label = new Label(studentTag);
+        switch (studentTag) {
+        case ("GOOD"):
```

Comment:
Very useful UI enhancement! Using red, green and yellow is quite an intuitive way to colour code the GOOD, AVERAGE and POOR tags